qualtran/bloqs/reflection_using_prepare.py

```diff
@@ -30,12 +30,17 @@
 class ReflectionUsingPrepare(GateWithRegisters):
     r"""Applies reflection around a state prepared by `prepare_gate`
 
-    Applies $R_{s} = I - 2|s><s|$ using $R_{s} = P(I - 2|0><0|)P^{\dagger}$ s.t. $P|0> = |s>$.
+    Applies $R_{s, g=1} = g (I - 2|s><s|)$ using $R_{s} = P(I - 2|0><0|)P^{\dagger}$
+    s.t. $P|0> = |s>$.
+
     Here
         $|s>$: The state along which we want to reflect.
         $P$: Unitary that prepares that state $|s>$ from the zero state $|0>$
         $R_{s}$: Reflection operator that adds a `-1` phase to all states in the subspace
             spanned by $|s>$.
+        $g$: The global phase to control the behavior of the reflection. For example:
```

Comment:
Maybe write this down like $R = g (I - 2|s><s|$)?

Reply:
done